src/views/dialog/script.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Dialog Script View Template
+ * The base script template for tribe-dialog.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe/dialogs/script.php
+ *
+ * @since TBD
+ *
+ * @package Tribe
+ * @version TBD
+ */
+?>
+<script>
+	var tribe = tribe || {};
+	tribe.dialogs = tribe.dialogs || [];
+
+	tribe.dialogs.push({
+		'appendTarget'         : '<?php echo esc_html( $append_target ); ?>',
```

Comment:
I wonder if we should set up an array of this information in PHP above, with the esc_html() usage, and then echo the wp_json_encode version into the `.push( ... )`. Would that simplify things and get rid of the potential single quote escaping problems currently present? It could also be... a filterable array

Reply:
THat's...a good idea!